helm/datalab/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.14.2
+version: 0.14.3
```

Comment:
I think this should now be 0.15 as it contains the infrastrucutre api changes that Andy has made